deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/FileSystemDeploymentService.java

```diff
@@ -152,6 +153,11 @@ class FileSystemDeploymentService implements DeploymentScanner, NotificationHand
     @SuppressWarnings("deprecation")
     private final DeploymentTransformer deploymentTransformer;
 
+    private ScanContext scanContext;
+
+    public Map<String,Boolean> getScanContextRegisteredDeployments() {
```

Comment:
This should be package-protected and moved down in the file to a less prominent position. Please add method javadoc stating the method only exists for testing.  The test is in the same package, so package-protected works. That kind of thing is why it's in the same package. :)